relm4-macros/src/attrs.rs

```diff
@@ -93,28 +120,65 @@ impl Parse for Attrs {
 
         let mut attrs = Attrs::new();
 
-        // Attrs::stream_peek(input.cursor());
-
-        if input.peek(Token![pub]) {
-            let pub_vis = if input.is_empty() {
-                None
-            } else {
-                Some(input.parse()?)
-            };
-            attrs.visibility.pub_vis = pub_vis;
-            attrs.visibility_set = true;
-        }
-        else if input.peek(Ident) && input.peek2(Token![=]){
-            let _ident: Ident = input.parse()?;
-            let _eq: Token![=] = input.parse()?;
-
-            let pub_vis = if input.is_empty() {
-                None
+        while !input.is_empty() {
+
+            // Attrs::stream_peek(input.cursor());
+
+            if input.peek(Token![pub]) {
+                if attrs.visibility_set {
+                    return Err(input.error("You can't assign visibility twice"));
+                }
+                let pub_vis = if input.is_empty() {
```

Comment:
This can never be `true`, right? Otherwise the `while` condition would be `false`. 

Also IMO we don't need backwards comparability. Using `visibility = pub` is just much clearer than `pub` especially since we can have multiple arguments now.

Reply:
> This can never be true, right? Otherwise the while condition would be false.

if you write `#[widget(visibility = pub, visibility = crate)] it can cause second assignment. It's stupid but having error here is cheap before somebody start to rely on undefined behavior (probably by mistake).

> Also IMO we don't need backwards comparability. Using visibility = pub is just much clearer than pub especially since we can have multiple arguments now.

I think rewriting relm4 due to reexports will be rather niche use case. While important for bigger projects. That's why I would keep option to use `#[widget(pub)]`. It's clean and compact. Most people would drift toward `visibility = pub` if they need to use `relm4` argument.

I propose the middle ground. If you use more then one argument you must use named args.